streamlit-python/local_test_env.yml

```diff
@@ -0,0 +1,12 @@
+# This file is used to install packages for local testing
+name: streamlit-python-testing
+channels:
+  - snowflake
+dependencies:
+  - python=3.8
+  - snowflake-snowpark-python
+  - pip
```

Comment:
added this due to warning from conda:
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.